Docs/user-guides/reference/02-working-with-openl-tables/03-table-types/02-decision-table/07-multiple-return-columns-in-smart-rules-tables.md

```
@@ -0,0 +1,21 @@
+###### Multiple Return Columns in Smart Rules Tables
+
+A smart rules table can contain up to three return columns. If the first return column contains a non-empty result, it is returned, otherwise, the next return column is scanned until the non-empty result is found or the last return column is verified.
+
+The following example illustrates a table with multiple return columns.
+
+![](../../../../ref_guide_images/6515353e9013232d853eff0f0fa13732.jpeg)
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)